lib/ansible/plugins/connection/docker.py

```diff
@@ -100,13 +109,23 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
             # This saves overhead from calling into docker when we don't need to
             self.actual_user = self._get_docker_remote_user()
 
+    def _split_cmd(self, argstring):
+        """Split a command string using shell semantics.
+
+        Shamelessly stolen from the SSH connection plugin.
+        """
+        return [
+            to_unicode(x.strip()) for x in shlex.split(to_bytes(argstring))
+            if x.strip()]
+
     @staticmethod
     def _sanitize_version(version):
         return re.sub('[^0-9a-zA-Z\.]', '', version)
 
     def _get_docker_version(self):
 
-        cmd = [self.docker_cmd]
+        # Force a quick copy, don't want to mutate base command.
```

Comment:
Can you explain this a bit more?  I don't think anything is wrong, I just don't understand  :)

Reply:
If the line would be `cmd = self.docker_cmd`, the original value of `self.docker_cmd` (which is a list) would be modified by the `cmd += ...`'s below. That's why a copy must be created.